### PR TITLE
normalize nuget version on FindPackageVersion

### PR DESCRIPTION
**Pull request by lktraser**
#3590

fixing FindPackageVersion for runtime packages with versions with trailing 0  in build or revision.

### CODE REVIEW

File: NuGet/NuGetFeedClass.ps1 [hunk 1]
```diff
@@ -277,7 +277,7 @@ class NuGetFeed {
             if ($excludeVersions -contains $version) {
                 continue
             }
-            if (($select -eq 'Exact' -and [NuGetFeed]::NormalizeVersionStr($nuGetVersionRange) -eq [NuGetFeed]::NormalizeVersionStr($version)) -or ($select -ne 'Exact' -and [NuGetFeed]::IsVersionIncludedInRange($version, $nuGetVersionRange))) {
+            if (($select -eq 'Exact' -and [NuGetFeed]::NormalizeVersionStr($nuGetVersionRange) -eq [NuGetFeed]::NormalizeVersionStr($version)) -or ($select -ne 'Exact' -and [NuGetFeed]::IsVersionIncludedInRange([NuGetFeed]::NormalizeVersionStr($version), [NuGetFeed]::NormalizeVersionStr($nuGetVersionRange)))) {
```
Comment (freddydk) [inline NuGet/NuGetFeedClass.ps1:280]: I don't think you can use the NormalizeVersionStr on the version range, which can be "[24.4.0.0)" - or?

**Review by freddydk — CHANGES_REQUESTED** [review]

File: NuGet/NuGetFeedClass.ps1 (hunk 1 above)
Comment (freddydk) [reply NuGet/NuGetFeedClass.ps1:280]: Looking in the IsVersionIncludedInRange function - that one already uses NormalizeVersionStr on the individual version numbers found.

Comment (lktraser) [reply NuGet/NuGetFeedClass.ps1:280]: I was wondering about that to but the first part of the line also used Normalization on the Range so I didn't look too hard at it.

Comment (lktraser) [reply NuGet/NuGetFeedClass.ps1:280]: Absolutely correct, I reran my tests to see if I just phantom fixed the issue with this and that seems to be the case. 
I'll abandon this.

**Pull request closed without merging**